### ISSUE TITLE
Solid background doesn't fill the whole area of the window

### ISSUE THREAD

**Issue by eyalsk** [cbucher/console#263]
I'm using Windows 10 x64 bit RTM.

The steps to reproduce it on my machine are quite simple as in downloading the program and extracting it and then use the default configuration that provided with it.

Here is how it looks when I expand it to full window.
![conzolez](https://cloud.githubusercontent.com/assets/3756641/9594281/da1cf548-5061-11e5-997a-502e8b5964d0.png)
Here is how it looks when I resize it.
![conzolez_small](https://cloud.githubusercontent.com/assets/3756641/9594287/eb1e6142-5061-11e5-9ac3-a4725cc89f85.png)
Here is the diagnostic output I got when I ran diagnose.
```text
ConsoleZ aero amd64 1, 14, 0, 15129
is elevated? no
UAC prefix "Administrator: "
Tab (active): ConsoleZ
  View (active): 1. ConsoleZ
  is elevated? no
font index: 10
face name: Consolas
font familly: 54 MODERN vector true type
font weight: 400
font size: width=9 height=20
max window size: cols=99 rows=50
Monitors 3
  Flags 1  primary
  DeviceID \\?\DISPLAY#SAM07D6#7&b09ba0f&0&UID12545#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}
  DeviceKey \Registry\Machine\System\CurrentControlSet\Control\Class\{4d36e96e-e325-11ce-bfc1-08002be10318}\0002
  DeviceName \\.\DISPLAY1\Monitor0
  DeviceString Generic PnP Monitor
  StateFlags 3
  Rect (0,0)x(1920,1080)
  Work (0,0)x(1920,1040)
  Flags 0
  DeviceID \\?\DISPLAY#SAM0659#7&b09ba0f&0&UID12547#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}
  DeviceKey \Registry\Machine\System\CurrentControlSet\Control\Class\{4d36e96e-e325-11ce-bfc1-08002be10318}\0003
  DeviceName \\.\DISPLAY2\Monitor0
  DeviceString Generic PnP Monitor
  StateFlags 3
  Rect (-3840,0)x(-1920,1080)
  Work (-3840,0)x(-1920,1080)
  Flags 0
  DeviceID \\?\DISPLAY#PHLC05F#7&b09ba0f&0&UID12544#{e6f07b5f-ee97-4a90-b076-33f57bf4eaa7}
  DeviceKey \Registry\Machine\System\CurrentControlSet\Control\Class\{4d36e96e-e325-11ce-bfc1-08002be10318}\0001
  DeviceName \\.\DISPLAY3\Monitor0
  DeviceString Generic PnP Monitor
  StateFlags 3
  Rect (-1920,0)x(0,1080)
  Work (-1920,0)x(0,1080)
Settings file internal resource
```

**Comment by cbucher** [comment]
The background is black and fill the entire window.

```
face name: Consolas
font size: width=9 height=20
max window size: cols=99 rows=50
```
The diagnostic shows you are limited to 99x50 chars because you set a too big font (Consolas with size **20**) in Windows console settings.
primary monitor height (1040) - caption (32) / font height (20) = max rows (50)

Fix this size (see #203).

**Comment by eyalsk** [comment]
I got it, thank you.
